QUARKUS-5186.md

```diff
@@ -0,0 +1,49 @@
+# QUARKUS-4573 - Enable Quarkus QE baremetal test execution of QuarkusIO/Quarkus project test coverage against RHBQ builds
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5186
+
+TD:
+- internal repo (Red Hat GitLab with Jenkins Jobs mainly)
+- https://github.com/quarkus-qe/quarkus-extracted-tests
+- https://github.com/quarkus-qe/quarkus-test-extractor
+
+[Quarkus main project](https://github.com/quarkusio/quarkus) has automated unit and integration test coverage that should verify functional and other requirements.
+Quarkus QE is tasked with Red Hat Build of Quarkus verification, which from practical POV means we need to have capability to run tests against delivered RHBQ artifacts.
+So far our team has tested the product with our own test coverage, however the Quarkus main project should be more exhaustive, and it has additional advantages, like that it is maintained by engineering teams.
+Delivered RHBQ bits should be 99.99 % same, therefore the fact that we only executed our tests were sufficient to detect a production process issues that would cause difference between community and product builds.
+The goal of this task is to extend the test coverage used for RHBQ verification by executing the Quarkus main project tests as well.
+The capability run these tests with RHBQ will avoid test development duplications on the QE side for tests run on bare-metal, cleanup Quarkus QE Test Suite, thus decrease maintenance and test execution time.
+We wish to follow with adding a capability to run the Quarkus main project tests in OpenShift and elaborate with engineering team on test requirements for noteworthy features 
```

Comment:
Add new empty line before this so the rendering looks better

Reply:
done